checkstyle.xml

```diff
@@ -8,7 +8,9 @@
 
     <module name="NewlineAtEndOfFile"/>
     <module name="Translation"/>
-    <module name="FileLength"/>
+    <module name="FileLength">
+        <property name="max" value="2100"/>
+    </module>
```

Comment:
We definitely don't need this change.
Refactor file, separate logic to different classes